features/step_definitions/branch_steps.rb

```diff
@@ -116,7 +116,12 @@
 
 
 Then(/^there are no more feature branches$/) do
-  expect(existing_branches).to match_array ['main', 'origin/main']
+  expected_branches = ['main', 'origin/main']
+  perennial_branches.delete_if { |b| b == '[none]' }.each do |perennial_branch|
```

Comment:
Can we add the delete_if to the `perennial_branches` function? We would need it anytime we use that function